tests/basic/legacy_shared_sharedv4_service_migration_test.go

```diff
@@ -531,8 +531,10 @@ var _ = Describe("{LegacySharedToSharedv4ServiceNodeDecommission}", func() {
 			for _, ctx := range contexts {
 				checkMapOfPods(podMap, ctx)
 			}
-			ValidateApplications(contexts)
+			// Don't validate apps.
```

Comment:
can you use this function PrereqForNodeDecomm for moving replicas before docomm

Reply:
Will address this in new ticket.